src/actions.js

```diff
@@ -151,3 +151,8 @@ export function authenticateAs(checker, action){
       .then(() => checker.driver.get(checker.lastUrl))
   })
 }
+
+export function scrollTo(checker, action){
+  const coord = action.scrollTo;
+  checker.driver.executeScript("window.scrollTo(0," + coord + ")");
```

Comment:
Your code only supports vertical scrolling, but please also support horizontal scrolling.

```js
{scrollTo: {x: 0, y: 100}}
```

I think that it is good to pass arguments as objects, and for axes omitted, set 0 to the default value.

Reply:
Please write test too.